packages/grid/_modules_/grid/hooks/virtualization/useVirtualRows.ts

```diff
@@ -206,6 +206,8 @@ export const useVirtualRows = (
   const scrollingTimeout = React.useRef<any>(0);
   const onScroll: any = React.useCallback(
     (event: any) => {
+      if (event.target.scrollLeft < 0) return;
+
       realScrollRef.current = { left: event.target.scrollLeft, top: event.target.scrollTop };
```

Comment:
The scroll event doesn't bubble, it might be clearer like this.

```suggestion
      realScrollRef.current = { left: windowRef.current!.scrollLeft, top: windowRef.current!.scrollTop };
```

Reply:
Done, even better, we don't need the `event` at all as I also used `windowRef.current!.scrollLeft` to fix the IOS bug.